src/commonMain/kotlin/kotools/types/number/AnyInt.kt

```diff
@@ -23,6 +23,56 @@ public sealed interface AnyInt {
     override fun toString(): String
 }
 
+/** Adds the [other] integer to this one. */
+@SinceKotoolsTypes("4.1")
+public operator fun Int.plus(other: AnyInt): Int = this + other.toInt()
+
+/** Adds the [other] integer to this one. */
+@SinceKotoolsTypes("4.1")
+public operator fun AnyInt.plus(other: Int): Int = toInt() + other
```

Comment:
We should keep the extension functions because of some clashes between declarations.
Here's an example for the `rem` operation for the `AnyInt` and the `PositiveInt` types:

```kotlin
operator fun AnyInt.rem(other: NonZeroInt): Int
operator fun PositiveInt.rem(other: NonZeroInt): PositiveInt
```

Reply:
But if we still want to convert extension functions to member functions, operations should return an `AnyInt` instead of an `Int`.
Here's an example for the `AnyInt.rem` operation above:
```kotlin
// before
operator fun AnyInt.rem(other: NonZeroInt): Int
// after
operator fun AnyInt.rem(other: NonZeroInt): AnyInt
```

Reply:
Here's a new checklist tracking the conversion of the following operations to member functions and their definition updating:
- [ ] `plus`
- [ ] `minus`
- [ ] `times`
- [ ] `div`
- [ ] `rem`.

Reply:
The issue #42 should be closed before resolving this conversation!

Reply:
With our minimalist approach, we want to keep only what is essential in the type definition. So additional features should be implemented with extension functions or properties. This is why we decide to not apply this conversion from extension to member function.